Makefile

```diff
@@ -122,3 +122,6 @@ report:
 	bisect-ppx-report -I _build -html $(COVERAGE) /tmp/bisect-xenops*out
 
 .PHONY: report
+
+reindent:
+	ocp-indent --inplace **/*.ml*
```

Comment:
Please use 
```
git ls-files '*.ml' '*.mli' | xargs ocp-indent --syntax cstruct -i
```
from https://github.com/xapi-project/xcp-idl/blob/master/Makefile

Reply:
Actually do we use the `cstruct` syntax extension anywhere? I only added it because I've seen it on one of the ring3 training examples.

Reply:
I am not sure, we were already using it somewhere. I copied it from one of our repositories when I made the example.